docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -305,6 +305,22 @@ In the example below:
 If you are dynamically switching the `leafField` or `mainGroupingCriteria`, the sorting and filtering models will not be cleaned up automatically, and the sorting/filtering will not be re-applied.
 :::
 
+## Propagate row selection
```

Comment:
```suggestion
## Automatic children selection
```

Reply:
I like the second one more.

The motivation behind naming the feature/flag `row selection propagation` is that it not only selects children on selecting parents but also selects, deselects, and puts into indeterminate mode the parents whose children have been selected or deselected. But from an SEO pov, mentioning the simpler use case (children selection) would be better. 👍